### DISCUSSION TITLE
what is wrong ?

### DISCUSSION THREAD

**Discussion by lj267lj267** [flet-dev/flet#3232] [Q&A]
### Question

lj@lj-virtual-machine:~/test$ flet build apk
Creating Flutter bootstrap project...OK
Customizing app icons and splash images...OK
Generating app icons...OK
Generating splash screens...OK
Packaging Python app...Flet app package app/app.zip was not created.

### Error message

```bash
lj@lj-virtual-machine:~/test$ flet build apk
Creating Flutter bootstrap project...OK
Customizing app icons and splash images...OK
Generating app icons...OK
Generating splash screens...OK
Packaging Python app...Flet app package app/app.zip was not created.
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Add `-vv` flag for verbosity.

**Reply by lj267lj267** [reply]
Thanks
